racetime/static/racetime/style/race.css

```diff
@@ -11,7 +11,7 @@ body {
     flex-direction: column;
 }
 
-.container > .content {
+.container>.content {
```

Comment:
Linter again is making a mistake here I think. Would prefer to keep these spaces between the `>` selector in.

Reply:
Fair point. Swapping it out. Gotta disable it in vs code. I'll just manually modify it via notepad++